tweakwcs/linearfit.py

```diff
@@ -207,16 +207,19 @@ def iter_linear_fit(xy, uv, wxy=None, wuv=None,
     """
     if fitgeom == 'general':
         linear_fit = fit_general
+        minobj = 3
```

Comment:
These checks replicate checks done in `align_wcs`.  Seems like overkill.  Would be better to do the check in the function that will fail if the check fails?